src/models/tag.ts

```diff
@@ -137,7 +137,7 @@ export class TagModel {
     return tag;
   }
 
-  public getBySaveId(id: string): Promise<Tag[]> {
+  public async getBySaveId(id: string): Promise<Tag[]> {
```

Comment:
thinking if this has to be `async` method as it returns promise.Tag and not having this somehow prioritizes resolving `tags` before completing the write